packages/shared/src/lib/core/wallet/actions/send/sendTransactionFromEvm.ts

```diff
@@ -27,6 +28,7 @@ export async function sendTransactionFromEvm(
     return new Promise((resolve, reject) => {
         checkActiveProfileAuth(
             async () => {
+                modifyPopupState({ preventClose: true })
```

Comment:
This is problematic for ledger devices, because it doesn't allow to open the VerifyLedgerTransaction popup anymore.